qa/rpc-tests/wallet_shieldingcoinbase.py

```diff
@@ -300,13 +301,14 @@ def run_test (self):
 
         # check balance
         node2balance = amount_per_recipient * num_t_recipients
-        saplingvalue -= node2balance + DEFAULT_FEE
+        saplingvalue -= node2balance + conventional_fee(2+num_t_recipients) # 2+ for padded Sapling input/change
         assert_equal(self.nodes[2].getbalance(), node2balance)
         check_value_pool(self.nodes[0], 'sapling', saplingvalue)
 
         # Send will fail because fee is negative
         try:
-            self.nodes[0].z_sendmany(myzaddr, recipients, 1, -1)
+            # NB: Using -2 as the fee because -1 is the sentinel for using the conventional fee.
+            self.nodes[0].z_sendmany(myzaddr, recipients, 1, -2)
```

Comment:
This can be reverted if we use `ZIP_317_FEE = None`.

Reply:
No it can't, because the relevant sentinel is the `-1` in the JSON-RPC (which takes fee as a number), so an explicit `-1` here would still be interpreted as `ZIP_317_FEE`.